src/main/java/modtrekt/logic/parser/ModtrektParser.java

```diff
@@ -62,6 +63,13 @@ public Command parseCommand(String userInput) throws ParseException {
                 .addCommand(UndoneModuleCommand.COMMAND_WORD, new UndoneModuleCommand())
                 .addCommand(AddModuleCommand.COMMAND_WORD, new AddModuleCommand())
                 .addCommand(AddModuleCommand.COMMAND_WORD_SHORTHAND, new AddModuleCommand())
+                .addCommand(CdModuleCommand.COMMAND_WORD, new CdModuleCommand())
+                .addCommand(RemoveCommand.COMMAND_WORD, new RemoveCommand())
+                .addCommand(RemoveCommand.COMMAND_WORD_SHORTHAND, new RemoveCommand())
+                .addCommand(RemoveCommand.COMMAND_WORD_SHORTHAND_2, new RemoveCommand())
+                .addCommand(RemoveCommand.COMMAND_WORD_SHORTHAND_3, new RemoveCommand())
```

Comment:
`addCommand` accepts any number of aliases at the end (and therefore also a `String[]`, see above), i.e. `addCommand(..., new RemoveCommand(), RemoveCommand.COMMAND_ALIASES...)`,
which cuts this down to just 1 line.